dir.props

```diff
@@ -86,6 +86,9 @@
     <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
+    <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/symreader/api/v3/index.json" />
+    <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/symreader-native/api/v3/index.json" />
+    <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/symreader-converter/api/v3/index.json" />
```

Comment:
We were using really old version of symreader that you could get from nuget.org.   To get the newer myget versions you need to add these feeds.

Reply:
I actually mirrored the packages on https://dotnet.myget.org/gallery/dotnet-buildtools to make sure old versions don't get dropped off the symreader feeds and break the BuildTools build. I don't think any versions of these packages are available on NuGet.org.

I've gone ahead and mirrored the new versions, too.